plugin.js

```diff
@@ -111,24 +119,37 @@
             var rect = DOM.getRect(this.win);
             var viewport = DOM.getViewPort(window);
 
-            DOM.setStyles(this.win,{
-                top:(pos.y +20) +'px',
-                left:Math.min(pos.x +10, Math.max(0,viewport.w-rect.w-20)) +'px'
+            var yPos = pos.y - jQuery(document).scrollTop() + 20;
+            DOM.setStyles(this.win, {
+                top: yPos + 'px',
+                left: Math.min(
+                    pos.x + 10,
+                    Math.max(0, viewport.w - rect.w - 20)) + 'px'
             });
         },
         _newWindow:function(id) {
-            var win = DOM.create('div',{
+            var win = DOM.create('div', {
                 'class':'mce_editorwindow',
                 'id':id
             });
-            this._addAll(win, ['div',{id:id+'_top'},
-                               ['a', {id:id+'_close','class' : 'mce_editorwindow_closebtn', tabindex : '-1', href : 'javascript:;', onmousedown : 'return false;'},'x'],
-                               ['div',{id:id+'_content'}]
-                              ]
-                        );
+            this._addAll(
+                win,
+                [
+                    'div',
+                    { id: id + '_top' },
```

Comment:
no space after { ?

(not sure if you're going for jshint compatibility in this pr. if not, please ignore)